src/win/fs-event.c

```diff
@@ -43,7 +43,7 @@ static void uv_fs_event_queue_readdirchanges(uv_loop_t* loop,
   if (!ReadDirectoryChangesW(handle->dir_handle,
                              handle->buffer,
                              uv_directory_watcher_buffer_size,
-                             FALSE,
+							               (handle->flags & UV_FS_EVENT_RECURSIVE) ? TRUE : FALSE,
```

Comment:
I don't think this is going to fly. UV_FS_EVENT_RECURSIVE has the same value as UV_HANDLE_ENDGAME_QUEUED, which is used internally by Windows handles. That's why the initial PR added a new field.